### DISCUSSION TITLE
QR code for payments doesn't work with Trust Wallet

### DISCUSSION THREAD

**Discussion by ilearnio** [btcpayserver/btcpayserver#4763] [Technical Support]
When I try to pay by scanning a QR code generated by BTCPay, using my iPhone that has Trust Wallet, I'm getting error in trust wallet "Deep Link is not supported". I figured that this happens because the QR code contains a link with a "pj" query argument, like `bitcoin:BC1Q5XFT6JT7PGUKZ7746Z2MSM3DY489DJPQTYGFEY?amount=0.00050029&pj=https://btcpay379576.lndyn.com/BTC/pj`

But, if I would generate a new QR code (using some online QR code generator tool) without that "pj" parameter then Trust Wallet understands such QR. 

Here's the QR code so you can test it yourself (use iOS device to scan it, I'm not sure if Android would also have such error):

![image](https://user-images.githubusercontent.com/14331216/224396352-249c6d44-dcab-4599-86e9-bbe7c2b12c63.png)

Another issue, is that "Open in wallet" button that is located below the QR code, when clicked redirects me to some BTCPay page, instead of opening a wallet app on my laptop. See screencast: https://take.ms/Ug9xX

These issues started to appear after I removed my BTC wallet from BTCPay admin and added the same one back (by entering 12 seed phrase)

**Comment by Kukks** [comment]
The link takes you to a btcpay page because you configured your browser to
handle bitcoin links with a btcpay store wallet.

We are using the bip21 spec for both the link and the qr. Trust wallet
needs to obviously support it properly instead of some naive
implementation. Any chance you can ping them to alert of this bug?

**Comment by ilearnio** [comment]
@Kukks sure, I will ping them about the error. If you're using a spec then I guess you're right, it should be fixed on Trust wallet end. Thank you so much for reply
